apps/designer/app/routes/designer/$id.tsx

```diff
@@ -54,14 +52,15 @@ export const action: ActionFunction = async ({ request, params }) => {
     );
 
     const imagesInfo = ImagesUpload.parse(formData.getAll("image"));
+
     imagesInfo.forEach((image) => {
       const data = {
-        type: image.type,
         name: image.name,
         path: `${path.join("/", folderInPublic, image.name)}`,
       };
+      const absolutePath = path.join(directory, image.name);
       const projectId = params.id as string;
-      db.assets.create(projectId, data);
+      db.assets.create(projectId, data, absolutePath);
```

Comment:
Anything uploaded will not send an absolute path since we can fetch this new image and get it's size